Builds/LinuxMakefile/Makefile

```diff
@@ -348,21 +347,9 @@ $(JUCE_OUTDIR)/$(JUCE_TARGET_VST) : $(OBJECTS_VST) $(RESOURCES) $(JUCE_OUTDIR)/$
 	-$(V_AT)mkdir -p $(JUCE_VSTDESTDIR)
 	-$(V_AT)cp -R $(JUCE_COPYCMD_VST)
 
-$(JUCE_OUTDIR)/$(JUCE_TARGET_VST3) : $(OBJECTS_VST3) $(RESOURCES) $(JUCE_OUTDIR)/$(JUCE_TARGET_SHARED_CODE)
```

Comment:
these parts in the Makefile gives errors, Is VST3 Relevant for Linux at all?

Reply:
[yes](https://forums.steinberg.net/t/linux-vst3/674195)

But I'd think this is a bug in Projucer

Reply:
> these parts in the Makefile gives errors, Is VST3 Relevant for Linux at all?

Why would VST3 not be "relevant" for Linux?